osx/Keybase.xcodeproj/project.pbxproj

```diff
@@ -610,6 +610,7 @@
 					001C6B361B24F8630048F086 = {
 						CreatedOnToolsVersion = 6.3.2;
 						DevelopmentTeam = 99229SGT5K;
+						ProvisioningStyle = Manual;
```

Comment:
@mmaxim  / @chrisnojima  --- do you guys have any insight into these project.pbxproj changes?

Reply:
i think we likely don't want manual

Reply:
I've reverted the project.pbxproj changes.  I guess they weren't needed -- I just needed the Mac Developer cert in my keychain.